pyproject.toml

```diff
@@ -1,3 +1,35 @@
+[build-system]
+requires = ["flit_core >=3.2,<4"]
```

Comment:
Just curious, is there a reason we're going with flit as a build backend and not something else? Can we not use the default build backend for some reason?

Reply:
I'd be curious if we can just use the default build backend here, but don't waste too much of your time if it seems like it doesn't work!

Reply:
Switched to `setuptools` -- looks like it works just fine, since it respects the already-checked-in `MANIFEST.in` 🙂